README.md

```diff
@@ -35,6 +35,26 @@ to your project or organization:
   command, because [changing default branch of a repository currently requires administrative token rights](https://developer.github.com/v3/repos/#update-a-repository).
   You can generate a token from your [personal access tokens page](https://github.com/settings/tokens/new).
 
+> ### Triggering release workflow events
+>
+> Because the tokens generated by GitHub Actions are considered OAuth tokens,
+> they are incapable of triggering further workflow events ([see this document for
+> an explanation](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#triggering-new-workflows-using-a-personal-access-token)).
+>
+> As such, if you want to trigger a release event when automatic-releases runs,
+> you will need to modify your `.github/workflows/release-on-milestone-closed.yml`
+> file to assign the `ORGANIZATION_ADMIN_TOKEN` as the value of the
+> `GITHUB_TOKEN` environment variable for the `Release` step:
+>
+> ```yaml
+>     - name: "Release"
+>       uses: "./"
+>       with:
+>         command-name: "laminas:automatic-releases:release"
+>       env:
+>         "GITHUB_TOKEN": ${{ secrets.ORGANIZATION_ADMIN_TOKEN }}
+> ```
+
 ## Usage
```

Comment:
I think `Usage` should come before the `Triggering release workflow events`, which is a bit of a niche

Reply:
Reasonable - will push in a moment.